### PR TITLE
Fix #278 CX truth table

### LINKED ISSUE #278: CX truth table
It seems '11' input is missing in the truth table.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5925baf7-4435-0027-80b7-8123de4c95a1
* Version Independent ID: e5004bc5-54da-b127-3021-df1b72924881
* Content: [CX operation (Microsoft.Quantum.Canon) - Q# reference - Microsoft Quantum](https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.canon.cx)
* Content Source: [api/qsharp/microsoft.quantum.canon.cx.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/live/api/qsharp/microsoft.quantum.canon.cx.yml)
* Product: **quantum**
* GitHub Login: @anpaz-msft

### PR DESCRIPTION
**Pull request by ihristova11**
Fixes #278

### CODE REVIEW

File: Standard/src/Canon/CommonGates.qs [hunk 1]
```diff
@@ -12,10 +12,12 @@ namespace Microsoft.Quantum.Canon {
...
     /// $$
     /// \begin{align}
-    ///     1 & 0 & 0 & 0 \\\\
-    ///     0 & 1 & 0 & 0 \\\\
-    ///     0 & 0 & 0 & 1 \\\\
-    ///     0 & 0 & 1 & 0
+    ///     \left(\begin{matrix}
+    ///         1 & 0 & 0 & 0 \\\\
+    ///         0 & 1 & 0 & 0 \\\\
+    ///         0 & 0 & 0 & 1 \\\\
```
Comment (msoeken) [inline Standard/src/Canon/CommonGates.qs:18]:
```suggestion
    ///         1 & 0 & 0 & 0 \\
    ///         0 & 1 & 0 & 0 \\
    ///         0 & 0 & 0 & 1 \\
```

**Review by msoeken — APPROVED** [review]
Thanks for your fix. I made one more change that I noticed.

File: Standard/src/Canon/CommonGates.qs (hunk 1 above)
Comment (cgranade) [reply Standard/src/Canon/CommonGates.qs]: @msoeken: Unfortunately, `\\\\` are currently required to delineate rows in matrices. Each `\\` gets turned by Markdown into a single `\`, so `\\\\` is interpreted by the LaTeX engine used in documentation (to wit, MathJax) as `\\`.

File: Standard/src/Canon/CommonGates.qs [hunk 2]
```diff
@@ -12,10 +12,12 @@ namespace Microsoft.Quantum.Canon {
...
     /// \begin{align}
-    ///     1 & 0 & 0 & 0 \\\\
-    ///     0 & 1 & 0 & 0 \\\\
-    ///     0 & 0 & 0 & 1 \\\\
-    ///     0 & 0 & 1 & 0
+    ///     \left(\begin{matrix}
+    ///         1 & 0 & 0 & 0 \\
+    ///         0 & 1 & 0 & 0 \\
+    ///         0 & 0 & 0 & 1 \\
+    ///         0 & 0 & 1 & 0
```
Comment (cgranade) [inline Standard/src/Canon/CommonGates.qs:19]:
```suggestion
    ///         1 & 0 & 0 & 0 \\\\
    ///         0 & 1 & 0 & 0 \\\\
    ///         0 & 0 & 0 & 1 \\\\
    ///         0 & 0 & 1 & 0
```

**Pull request merged**